can/interfaces/usb2can/usb2canInterface.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         channel=None,
         dll="usb2can.dll",
         flags=0x00000008,
-        *args,
+        *_,
```

Comment:
Optional style change, this documents that all additional positional arguments get ignored but won't throw an exception when passed in.